src/counterfactuals/search.jl

```diff
@@ -20,7 +20,18 @@ function update!(ce::CounterfactualExplanation)
     ce.search[:times_changed_features] += _times_changed        # update number of times feature has been changed
     ce.search[:iteration_count] += 1                            # update iteration counter   
     ce.search[:path] = [ce.search[:path]..., ce.s′]
-    return terminated(ce)
+end
+
+function update!(ce::CounterfactualExplanations)
+    #TODO implement ce.search updating
+
+    if (ce.generator == :shrink)
+        growing_spheres_shrink!(ce)
+    elseif (ce.generator == :expand)
+        growing_spheres_expand!(ce)
+    else (ce.generator == :feature_selection)
+        feature_selection!(ce)
+    end
```

Comment:
@pat-alt @VincentPikand @RaunoArike do you guys have any better idea on how to implement `update!()` for growing spheres?

Reply:
Hi @kmariuszk, sorry I'm only now getting to this, been a very hectic week. Could a set up like in DiCE be applicable where multiple counterfactuals are produced at once?